src/main/java/org/broadinstitute/hellbender/tools/copynumber/arguments/GermlineDenoisingModelArgumentCollection.java

```diff
@@ -0,0 +1,159 @@
+package org.broadinstitute.hellbender.tools.copynumber.arguments;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.tools.copynumber.GermlineCNVCaller;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class GermlineDenoisingModelArgumentCollection implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    public enum CopyNumberPosteriorExpectationMode {
+        MAP("map"),
+        EXACT("exact"),
+        HYBRID("hybrid");
+
+        final String pythonArgumentString;
+
+        CopyNumberPosteriorExpectationMode(final String pythonArgumentString) {
+            this.pythonArgumentString = pythonArgumentString;
+        }
+    }
+
+    @Argument(
+            doc = "Maximum number of bias factors.",
+            fullName = "max-bias-factors",
```

Comment:
Extract these to `public static final` in this class.

Reply:
Done.